scripts/zones/Dynamis-Bastok/mobs/Vanguard_Undertaker.lua

```diff
@@ -4,6 +4,14 @@
 -----------------------------------
```

Comment:
SMN scripts can be reduced to:

```
mixins = {require("scripts/mixins/job_special")}

function onMobSpawn(mob)
    mob:setLocalVar("mainSpec", dsp.jsa.ASTRAL_FLOW_MAAT)
end

function onMobDeath(mob, player, isKiller)
end

function onMobDespawn(mob)
    DespawnMob(mob:getID() + 1)
end
```

This will make the mob use skill 1023 instead of the default ASTRAL_FLOW (734)